app/assets/stylesheets/color_themes/_color_theme_override.scss

```diff
@@ -1,18 +1,5 @@
 /* Raw CSS */
 body {
-  a,
-  a.tag,
-  .btn-link,
-  #main_stream .stream-element > .media a.author-name,
-  #hovercard h4 a,
-  .stream-element .from a.self {
-    color: $link-color;
-
-    &:hover, &:focus {
-      color: darken($link-color, 10%);
-    }
```

Comment:
This is still used in the themes but it looks different in the original, original white and dark theme. So removing it unifies the themes. 👍